ui/src/app/views/sign-in/component.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import {Button} from 'app/components/buttons';
+
+import {mount} from 'enzyme';
+import * as React from 'react';
+
+import SignInReact, {pageImages, SignInProps} from './component';
+
+describe('SignInReact', () => {
+  let props: SignInProps;
+
+  const signIn = jest.fn();
+
+  const component = () => {
+    return mount(<SignInReact {...props}/>);
+  };
+
+  beforeEach(() => {
+    props = {
+      onInit: jest.fn(),
```

Comment:
If you don't use this, I'd just give an empty arrow function rather than creating a mock